src/autopas/options/IteratorBehavior.h

```diff
@@ -58,20 +89,26 @@ class IteratorBehavior : public Option<IteratorBehavior> {
 
   /**
    * Set of options that are very unlikely to be interesting.
+   * Technically these options are nor discouraged but when fetching a set of desired behaviors these are probably
+   * not useful and would be excluded anyway.
    * @return
    */
-  static std::set<IteratorBehavior> getDiscouragedOptions() { return {}; }
+  static std::set<IteratorBehavior> getDiscouragedOptions() {
+    return {IteratorBehavior::dummy, IteratorBehavior::ownedOrHaloOrDummy, IteratorBehavior::forceSequential};
+  }
 
   /**
    * Provides a way to iterate over the possible choices of AcquisitionFunction.
    * @return map option -> string representation
    */
   static std::map<IteratorBehavior, std::string> getOptionNames() {
     return {
-        {IteratorBehavior::ownedOnly, "ownedOnly"},
-        {IteratorBehavior::haloOnly, "haloOnly"},
-        {IteratorBehavior::haloAndOwned, "haloAndOwned"},
-        {IteratorBehavior::haloOwnedAndDummy, "haloOwnedAndDummy"},
+        {IteratorBehavior::owned, "owned"},
+        {IteratorBehavior::halo, "halo"},
+        {IteratorBehavior::ownedOrHalo, "ownedOrHalo"},
+        {IteratorBehavior::dummy, "dummy"},
+        {IteratorBehavior::ownedOrHaloOrDummy, "ownedOrHaloOrDummy"},
+        {IteratorBehavior::forceSequential, "forceSequential"},
     };
   }
 
```

Comment:
possibly, it makes sense to add convenience functions to the behavior enum, such as:
```c++
doesIterateOwned();
goesIterateHalo();
```
I'm not happy with the names, though ^^

Reply:
I'm a bit against explicitly creating state checker functions for every single value. However, I could imagine a function like:
```
bool contains(IteratorBehavior behavior) {
  return _value & behavior;
}